src/components/Toggle/Toggle.jsx

```diff
@@ -63,6 +65,7 @@ class Toggle extends React.Component<Props> {
               className={toggleClasses}
               key={index}
               type={this.props.type}
+              data-state={toggle.disabled ? 'disabled' : 'enabled'}
```

Comment:
Don't see a necessity for this, but OK.

Reply:
there's no indication for _disabled_ items (expect of searching for "disabled" in the generated class name). This should help us testing.